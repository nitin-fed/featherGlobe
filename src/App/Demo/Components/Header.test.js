import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

describe('Header', () => {
  test('Should render', () => {
    const component = renderer.create(<Header />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
