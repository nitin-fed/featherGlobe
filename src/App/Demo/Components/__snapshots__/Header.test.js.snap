// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render 1`] = `
<div
  className="App-header"
>
  Application Header
  <span>
    Counter: 
    1
  </span>
</div>
`;
